tests/test_plot_interface.py

```diff
@@ -14,10 +14,20 @@
 def pfplot(pf_app):
     # Return PFPlotInterface instance and show app
     pfplot = powfacpy.PFPlotInterface(pf_app)
+    
+    return pfplot 
+
+@pytest.fixture(autouse=True)
+def run_around_tests(pfplot):
+    # Code run before each test:
     # PF app needs to be shown for some reason, because otherwise 
     # PF can error when plotting.
     pfplot.app.Show()
-    return pfplot 
+    # Run test
+    yield
```

Comment:
```suggestion
    yield # gives control back to the 'context manager', i. e. the function that called this fixture. 
```

If I understand correctly. 
yield is kind of uncommon, so I thought an explanatory comment could help.